install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
+system resources.
+
+[IMPORTANT]
+====
+The Prometheus is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[openshift-prometheus-roles]
+=== Setting Prometheus Role Variables
+
+The Prometheus role creates:
+
+* OpenShift-Prometheus namespace.
+* Prometheus clusterrolebinding and service account.
+* PVs for Prometheus, alertmanager and alertbuffer for internal NFS.
+* Prometheus pod with Prometheus behind oauth-proxy, alertmanager and alert-buffer behind oauth-proxy.
+* Prometheus and alertmanager configmaps.
+* Prometheus and alerts services and direct routes.
+* Prometheus, alertmanager and alert-buffer PVCs.
+
+Depends on `openshift_hosted_prometheus_deploy` flag.
+
+Set the following role variables to install and configure Prometheus.
+
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_state`
+|Set variable to install and update. Remove variable to uninstall.
+
+|`openshift_prometheus_namespace`
+|Project namespace where the components will be deployed. Default set to
+`openshift-prometheus`.
+
+|`openshift_prometheus_replicas`
+|The number of replicas for prometheus deployment. The default is one. Multiple
+replicas are not supported.
+
+|`openshift_prometheus_node_selector`
+|Selector for the nodes on which prometheus will be deployed.
+
+|`openshift_prometheus_image_proxy`
+|Set to specify image for the component. For example, "openshift/oauth-proxy:v1.0.0".
+
+|`openshift_prometheus_image_prometheus`
+|Set to specify image for the component. For example, "openshift/prometheus:v2.0.0-dev".
+
+|`openshift_prometheus_image_alertmanager`
+|Set to specify image for the component. For example, "openshift/prometheus-alertmanager:dev".
+
+|`openshift_prometheus_image_alertbuffer`
+|Set to specify image for the component. For example, "ilackarms/message-buffer".
+
+|`openshift_prometheus_additional_rules_file`
+|Additional Prometheus rules file. Set to `null` by default.
+
+|===
+
+[openshift-prometheus-deploy]
+=== Deploying Prometheus Using Ansible Installer
+
+The Ansible Installer is the default method of deploying Prometheus.
+
+Add label to your node:
+----
+region=infra and deploy prometheus via ansible with default values
+# default value inventory file
+openshift_prometheus_state=present
+openshift_prometheus_namespace=prometheus
+
+openshift_prometheus_replicas=1
+openshift_prometheus_node_selector={"region":"infra"}
+
+# images
```

Comment:
design is changed, we don't have the following parameters
openshift_prometheus_image_proxy
openshift_prometheus_image_prometheus
openshift_prometheus_image_alertmanager
openshift_prometheus_image_alertbuffer

you can contact the developers to get the details